src/pg/replication.cr

```diff
@@ -0,0 +1,353 @@
+module PG::Replication
```

Comment:
I can't tell yet if this should be `LogicalReplication` or if physical replication uses the same concepts.